src/js/themes/base.d.ts

```diff
@@ -1073,6 +1073,19 @@ export interface ThemeType {
       color?: ColorType;
     };
   };
+  page?: {
+    [key: string]: {
```

Comment:
I'm not sure if the ? is needed, but just wondering

```suggestion
    [key: string]?: {
```

Reply:
Nope, it causes an error when added